tests/std/tests/VSO_0000000_vector_algorithms/test.cpp

```diff
@@ -777,6 +837,11 @@ void test_swap_ranges(mt19937_64& gen) {
 }
 
 void test_vector_algorithms(mt19937_64& gen) {
+    test_adjacent_difference<int>(gen);

```

Comment:
⚠️  I want to highlight that I omitted 8 and 16 bit cases, as I'm not sure what to do with narrowing warnings ⚠️

Reply:
I can fix this easily.